lib/dslash_policy.cuh

```diff
@@ -336,11 +337,9 @@ namespace quda
 
   template <typename Dslash> struct DslashPolicyImp {
 
-    virtual void operator()(
-        Dslash &dslash, cudaColorSpinorField *in, const int volume, const int *faceVolumeCB, TimeProfile &profile)
-        = 0;
```

Comment:
Why we do not want this to be pure virutal anymore?

Reply:
This was a consequence of making the factory return a `unique_ptr`.  Not a big deal, since if this method isn't implemented then one will get a compilation error.